tests/integration/test_analysisd/test_eps/test_start_queuing_events_when_limit_reached.py

```diff
@@ -0,0 +1,111 @@
+@pytest.mark.parametrize('configure_local_internal_options_eps', [timeframe_eps_t1], indirect=True)
+@pytest.mark.parametrize('simulate_agent', [params_start_queuing_events_when_limit_reached], indirect=True)
+def test_start_queuing_events_when_limit_reached(configuration, metadata, set_wazuh_configuration_eps,
+                                                 truncate_monitored_files, restart_wazuh_daemon_function,
+                                                 simulate_agent):
+    '''
+    description: Check that the `events_processed` value in the `/var/ossec/var/run/wazuh-analysisd.state` file must
+                 be lower or equal than `maximum` * `timeframe` and, the `events_received` value must be greater than
+                 `events_processed` and, the `events_dropped` value equal to 0 and finaly, `event_queue_usage` is lower
+                 than 1.0.
+
+    test_phases:
+        - Set a custom Wazuh configuration.
+        - Truncate logs files.
+        - Restart wazuh-daemons.
+        - Execute agent simulated script.
+
+    wazuh_min_version: 4.4.0
+
+    tier: 0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Wazuh configuration data. Needed for set_wazuh_configuration fixture.
+        - metadata:
+            type: dict
+            brief: Wazuh configuration metadata.
+        - set_wazuh_configuration_eps:
+            type: fixture
+            brief: Set the wazuh configuration according to the configuration data.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - restart_wazuh_daemon_function:
+            type: fixture
+            brief: Restart all the wazuh daemons.
+        - simulate_agent:
+            type: fixture
+            brief: Execute a script that simulate agent and send `logcolector` logs to the manager.
+
+    assertions:
+        - The `events_processed` value in the `/var/ossec/var/run/wazuh-analysisd.state` file must be lower or equal
+          than `maximum` * `timeframe` and greater than a percentage of `maximum` * `timeframe` to confirm that
+          `events_processed` is not null. The `events_received` value must be greater than `events_processed` and,
+          the `events_dropped` value equal to 0 and finaly, `event_queue_usage` is lower than 1.0.
+
+    input_description:
+        - The `cases_start_queueing_events.yaml` file provides the module configuration for this test.
+    '''
+    # Wait 'timeframe' / 2 second to read the wazuh-analysisd.state to ensure that has corrects values
+    sleep(metadata['timeframe'] / 2)
+    events_processed = evm.get_analysisd_state('events_processed')
+    events_received = evm.get_analysisd_state('events_received')
+    events_dropped = evm.get_analysisd_state('events_dropped')
+    event_queue_usage = evm.get_analysisd_state('event_queue_usage')
```

Comment:
Is better to read all the analysisd values, and access desired values by map keys.